src/test/kotlin/uk/gov/dluhc/notificationsapi/testsupport/testdata/dto/TemplatePersonalisationDtoBuilder.kt

```diff
@@ -478,7 +478,8 @@ fun buildBespokeCommPersonalisationMapFromDto(
         personalisationMap["explainFreeText"] = whatToDo ?: ""
         personalisationMap["whatYouNeedToDo"] = whatToDo != null || deadline != null
         personalisationMap["deadline"] = deadline ?: ""
-        personalisationMap["sourceType"] = personalisationSourceTypeString
+        personalisationMap["an"] = personalisationFullSourceTypeString == "overseas vote"
+        personalisationMap["sourceType"] = personalisationFullSourceTypeString
```

Comment:
Its very simple so not really an issue, but this logic is duplicated in src/main/kotlin/uk/gov/dluhc/notificationsapi/mapper/TemplatePersonalisationDtoMapper.kt
Is there a way to avoid that? (very happy with the answer being no its not worth it but just want to check)

Reply:
ahh missed that this was for testing! that makes sense, I would say if its just a builder it should just take a hard coded value instead of having logic like this, but don't feel strongly about it so happy for it to stay as is